apps/console/src/features/identity-providers/components/settings/authenticator-settings.tsx

```diff
@@ -18,7 +18,7 @@
 
 import { AlertLevels, TestableComponentInterface } from "@wso2is/core/models";
 import { addAlert } from "@wso2is/core/store";
-import { ConfirmationModal, ContentLoader, EmptyPlaceholder, PrimaryButton } from "@wso2is/react-components";
+import { ConfirmationModal, ContentLoader, EmptyPlaceholder, PrimaryButton, SegmentedAccordionTitleActionInterface } from "@wso2is/react-components";
```

Comment:
Possible max-length issue here.

Reply:
Fixed this linting issue in commit 0094cf8